resources/github-release-script.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# Based on the shell script of Github user Jaskaranbir (https://gist.github.com/Jaskaranbir/d5b065173b3a6f164e47a542472168c1)
+
+# ===> Set these variables first
+branch="master"
+repo_slug="$TRAVIS_REPO_SLUG"
+token="$GITHUB_TOKEN"
+version="$TRAVIS_TAG"
+
+LAST_REVISION=$(git rev-list --tags --skip=1 --max-count=1)
+LAST_RELEASE_TAG=$(git describe --abbrev=0 --tags ${LAST_REVISION})
+
+# Generate CHANGELOG.md
+github_changelog_generator \
+  -u $(cut -d "/" -f1 <<< $repo_slug) \
+  -p $(cut -d "/" -f2 <<< $repo_slug) \
+  --token $token \
+  --since-tag ${LAST_RELEASE_TAG} \
+  --no-author \
+  --no-unreleased \
+  --header-label "## Changelog"
+
+sed -i -e '3,5d' CHANGELOG.md
+sed -i '$ d' CHANGELOG.md
+
+body="$(cat CHANGELOG.md)"
```

Comment:
Maybe you could add an additional header and a sentence where you link to the online version of the documentation prior to the changelog?

Reply:
Done.